spec/fixtures/csv/BlackpoolGazetteHerald.csv

```diff
@@ -0,0 +1,2 @@
+id,date_uploaded,date_modified,visibility,visibility_statement,visibility_statement.1,embargo_end_date,visibility_after_embargo,lease_end_date,visibility_after_lease,collection_id_1,collection_id_2,collection_id_3,work_type,resource_type,title,creator_name_type_1,creator_family_name_1,creator_given_name_1,creator_organisation_name_1,creator_isni_1,creator_orcid_1,creator_staffmember_1,creator_researchassociate_1,creator_name_type_2,creator_family_name_2,creator_given_name_2,creator_organisation_name_2,creator_isni_2,creator_orcid_2,creator_staffmember_2,creator_researchassociate_2,creator_name_type_3,creator_family_name_3,creator_given_name_3,creator_organisation_name_3,creator_isni_3,creator_orcid_3,creator_staffmember_3,creator_researchassociate_3,contributor_name_type_1,contributor_type_1,contributor_family_name_1,contributor_given_name_1,contributor_organisation_name_1,contributor_isni_1,contributor_orcid_1,contributor_staffmember_1,contributor_researchassociate_1,contributor_name_type_2,contributor_type_2,contributor_family_name_2,contributor_given_name_2,contributor_organisation_name_2,contributor_isni_2,contributor_orcid_2,contributor_staffmember_2,contributor_researchassociate_2,contributor_name_type_3,contributor_type_3,contributor_family_name_3,contributor_given_name_3,contributor_organisation_name_3,contributor_isni_3,contributor_orcid_3,contributor_staffmember_3,contributor_researchassociate_3,abstract,date_published,material_media_1,duration,institution_1,organisational_unit_1,organisational_unit_2,project_name,funder_1,funder_project_reference_1,event_title_1,event_location,event_date_1,series_name_1,book_title,editor_name_type_1,editor_family_name_1,editor_given_name_1,editor_organisation_name_1,editor_isni_1,editor_orcid_1,editor_staffmember_1,editor_researchassociate_1,editor_name_type_2,editor_family_name_2,editor_given_name_2,editor_organisation_name_2,editor_isni_2,editor_orcid_2,editor_staffmember_2,editor_researchassociate_2,journal_title,alternative_journal_title,volume_1,edition,version_1,issue,pagination,article_number_1,publisher_1,place_of_publication_1,isbn,issn,eissn,current_he_institution_name,date_accepted,date_submitted,official_url,related_url_1,related_url_2,related_exhibition_1,related_exhibition_date_1,related_exhibition_venue_1,language_1,license_1,rights_statement,rights_holder_1,doi,qualification_name,qualification_level,alternate_identifier_1,alternate_identifier_type_1,related_identifier_id_1,related_identifier_type_1,related_identifier_relationship_1,peer_reviewed,keyword_1,keyword_2,keyword_3,dewey_classification,library_of_congress_classification_1,library_of_congress_classification_2,additional_information

```

Comment:
diff here is just changing `current_he_institution` to `current_he_institution_name`